radar/2024-12-01/dlt.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Delta Live Tables (DLT)"
+ring: trial
+quadrant: languages-and-frameworks
+featured: true
+---
+
+[Delta Live Tables (DLT)](https://docs.databricks.com/delta-live-tables/index.html) is a declarative framework for building reliable, maintainable, and testable data processing pipelines. It simplifies ETL development by using a simple declarative approach to build and manage data pipelines on the Databricks platform.
```

Comment:
Is this link correct? Because it redirects to https://docs.databricks.com/aws/en/dlt which is about `Lakeflow Declarative Pipelines`. Or did they rename `Delta Live Tables` to `Lakeflow Declarative Pipelines` maybe? In that case I guess we should update this blip accordingly?

Makes good sense to me to use this in any case. :-)